nessie/lib/util.py

```diff
@@ -146,9 +146,9 @@ def resolve_sql_template_string(template_string, **kwargs):
         'loch_s3_canvas_data_path_current_term': s3_prefix + app.config['LOCH_S3_CANVAS_DATA_PATH_CURRENT_TERM'],
         'loch_s3_coe_data_path': s3_prefix + get_s3_coe_daily_path(),
         'loch_s3_physics_data_path': s3_prefix + app.config['LOCH_S3_PHYSICS_DATA_PATH'],
-        'loch_s3_sis_advising_notes_data_path': s3_advising_notes_prefix + app.config['LOCH_S3_SIS_ADVISING_NOTES_DATA_PATH'],
         'loch_s3_sis_data_path': s3_prefix + app.config['LOCH_S3_SIS_DATA_PATH'],
         'loch_s3_sis_api_data_path': s3_prefix + get_s3_sis_api_daily_path(),
+        'loch_s3_sis_data_protected_path': s3_advising_notes_prefix + app.config['LOCH_S3_SIS_DATA_PATH'],
```

Comment:
@lyttam Should we perhaps change the variable name `s3_advising_notes_prefix` (and the config value `LOCH_S3_SIS_ADVISING_NOTES_BUCKET`) to reflect their new function as all protected, all the time?

Reply:
@paulkerschen I thought about that too, though it would make more work for Darlene. But I suppose we'll want to change it eventually.

Reply:
@paulkerschen amended with your suggestion. I'll update nessie-dev.py as well.